docs/docs/tutorial/part-1/index.mdx

```diff
@@ -106,9 +117,27 @@ What would you like to name the folder where your site will be created?
 ✔ Desktop/ my-first-gatsby-site
 ```
 
-5. When the prompt asks, **"Will you be using a CMS?"** select **"No (or I'll add it later)"**.
+5. When the prompt asks, **"Will you be using JavaScript or TypeScript?"** choose between JavaScript or TypeScript.
+
+```shell
+? Will you be using JavaScript or TypeScript?
+(Single choice) Arrow keys to move, enter to confirm
+  JavaScript
+❯ TypeScript
+```
+
+<Announcement style={{marginBottom: "1.5rem"}}>
+
+This question will be skipped if you passed in a `-ts` flag at the beginning i.e `gatsby new -ts`.
+This indicates that you want to use TypeScript hence our [Minimal Typescript Starter](https://github.com/gatsbyjs/gatsby/tree/master/starters/gatsby-starter-minimal-ts) will be pulled in as the template.
```

Comment:
This announcement should point out that people can use TypeScript but if the want to follow the tutorial step-by-step they need to use JavaScript (as the tutorial uses that)

Reply:
👍